### ISSUE TITLE
Tag stays unconfirmed

### ISSUE THREAD

**Issue by liukkone** [akx/hass-ruuvitag#12]
Tag stays unconfirmed and doesn't appear anywhere in hass.io.  
Temperature and humid data is correct at the addon's log so it works allmost.

Home Assistant 0.101.3 / HassOS 2.12

`This is hassio-ruuvitag 0.2.0, terrrrrrrve
Found tag f3b91326c1b6
Found an unconfigured tag f3b91326c1b6. This will only be shown once per tag.
To help you identify this tag, its current information follows.
  {"dataFormat":3,"rssi":-81,"humidity":34,"temperature":22.78,"pressure":102931,"accelerationX":-19,"accelerationY":46,"accelerationZ":1009,"battery":2953}
To have its status posted to Home Assistant, add the following to the tags configuration:
  {"id":"f3b91326c1b6","name":"some-name","enabled":true,"temperature":true,"pressure":true,"humidity":true,"battery":true,"acceleration":false,"accelerationX":false,"accelerationY":false,"accelerationZ":false}`

**Comment by akx** [comment]
Did you add that object to the tags configuration, per https://github.com/akx/hass-ruuvitag#usage ?

**Comment by liukkone** [comment]
Yes, I edited config.json file and added ID, name, enabled etc. information to "tags".
Do I need to add something else to somewhere?
